woocommerce/woocommerce-android
# [Woo POS - payments onboarding] Remove local FF

Closes: #12928


### Description

The PR removes feature flag

### Testing information

* Use both release and debug builds
* Use site with unfinished onboarding
* More -> Payments 
* Notice that POS is available for both builds
* Notice that onboarding is shown on the both builds

### The tests that have been performed

Above
### Images/gif


https://github.com/user-attachments/assets/650dbb71-0b27-4356-9649-f8c8edc8cb49



## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
